frontend/app/error.vue

```diff
@@ -13,14 +13,20 @@ onMounted(() => {
 })
 
 const errorMessage = computed(() => {
-  switch (props.error?.statusCode) {
-    case 404:
+  const error = props.error
+  if (!error) {
+    return 'Nieznany błąd'
+  }
```

Comment:
You don't have to change that, but we are using K&R braces style, so you could write this like:
```ts
  if (!error)
    return 'Nieznany błąd'
```
or (but I am not sure if eslint will allow for that)
```ts
  if (!error) return 'Nieznany błąd'
```